src/base/base.pri

```diff
@@ -1,4 +1,5 @@
 HEADERS += \
+    $$PWD/algorithm.h \
```

Comment:
if possible you should also add an entry to CMakeLists.txt

Reply:
I'm working on updating the CMake stuff. Actually in the process I removed all the `.h` suff.
As I did not finished it, yet. Will add it to CMake.